build/squire-raw.js

```diff
@@ -1325,6 +1325,16 @@ var keyHandlers = {
             return;
         }
 
+        if ( /^PRE|CODE|SAMP$/.test( block.nodeName ) ) {
```

Comment:
`CODE` and `SAMP` are both inline node types (see Node.js), so can never be returned as the block type for getStartBlockOfRange. This test should just look for `block.nodeName === 'PRE'`